relays/substrate-client/src/client.rs

```diff
@@ -36,6 +34,7 @@ use sp_version::RuntimeVersion;
 use std::ops::RangeInclusive;
 
 const SUB_API_GRANDPA_AUTHORITIES: &str = "GrandpaApi_grandpa_authorities";
+const MAX_SUBSCRIPTION_CAPACITY: usize = 4096;
```

Comment:
What does this control exactly? Is it the maximal number of open subscriptions or pending notifications in the sbuscription?

Reply:
Pending notifications, https://github.com/paritytech/jsonrpsee/blob/master/types/src/client.rs#L66-#L70